.github/workflows/linux.yml

```diff
@@ -274,3 +274,88 @@ jobs:
           python3 -m pytest tokenizers_test.py
         working-directory: ${{ env.OPENVINO_TOKENIZERS_REPO }}/tests
 
+  openvino_tf_tests:
+    name: OpenVINO TensorFlow tests (BUILD_FAST_TOKENIZERS=${{ matrix.build_fast_tokenizers }})
+    strategy:
+      matrix:
+        build_fast_tokenizers: [ON, OFF]
+    needs: [ openvino_download, openvino_tokenizers_wheel]
+    if: always() && needs.openvino_tokenizers_wheel.result == 'success'
+    timeout-minutes: 45
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-22.04
+
+    env:
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/openvino/install
+      OPENVINO_TOKENIZERS_REPO: ${{ github.workspace }}/openvino_tokenizers
+      BUILD_DIR: ${{ github.workspace }}/openvino_tokenizers/build
+
+    steps:
+      - name: Clone Openvino TensorFlow tests
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: ${{ env.OPENVINO_REPO }}
+          sparse-checkout: |
+            tests/layer_tests
+
+      - name: Setup Python ${{ env.PYTHON_VERSION }}
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pip'
+
+      - name: Download tokenizers package
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: openvino_tokenizers_wheel_${{ matrix.build_fast_tokenizers }}
+          path: ${{ env.INSTALL_DIR }}/ov_tokenizers
+
+      - name: Download OpenVINO package
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: ${{ needs.openvino_download.outputs.ov_artifact_name }}
+          path: ${{ env.INSTALL_DIR }}
+          merge-multiple: true
+
+
+      - name: Install OpenVINO Python wheel from pre-built artifacts
+        run: |
+          python3 -m pip install openvino==${{ needs.openvino_download.outputs.ov_version }} ${{ needs.openvino_download.outputs.ov_wheel_source }}
+        working-directory: ${{ env.INSTALL_DIR }}
+
```

Comment:
also, we need to install these deps: https://github.com/openvinotoolkit/openvino/blob/master/tests/requirements_tensorflow